cvat/apps/engine/migrations/0036_auto_20201216_0943.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 3.1.1 on 2020-12-16 09:43
+
+import cvat.apps.engine.models
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('engine', '0035_data_storage'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='task',
+            name='dimension',
+            field=models.CharField(choices=[('3d', 'DIM_3D'), ('2d', 'DIM_2D')], default=cvat.apps.engine.models.DimensionType['DIM_2D'], max_length=2),
```

Comment:
from cvat.apps.engine.models import DimensionType

Use DimensionType.choices() method here.

Reply:
This is an auto-generated file, do we have to modify to as requested, kindly confirm.